examples/loopback/src/main.rs

```diff
@@ -0,0 +1,35 @@
+use std::io::{Read, Write};
+use std::net::{SocketAddr, TcpListener, TcpStream};
+use std::thread;
+
+#[cfg(target_os = "hermit")]
+use hermit as _;
+
+const TO_SEND: &[u8] = b"hello loopback";
+
+fn handle_client(mut stream: TcpStream) {
+	let mut buf = [0u8; TO_SEND.len()];
+	stream.read_exact(&mut buf).unwrap();
+	assert_eq!(&buf, TO_SEND);
+	stream.write_all(TO_SEND).unwrap();
+}
+
+// Requires HERMIT_IP=127.0.0.1
+
+fn main() {
```

Comment:
Can we return `io::Result<()>` here and at `handle_client` and `?` instead of unwrap?